sched/Kconfig

```diff
@@ -1162,6 +1162,13 @@ config SCHED_CPULOAD_TIMECONSTANT
 		tick count exceeds this time constant.  This time constant is in
 		units of seconds.
 
+config SCHED_PROFIL_TICKSPERSEC
```

Comment:
Same here, PROFILE instead of PROFIL

Reply:
ditto

Reply:
Yes, I think here in the menuconfig PROFILE name makes more sense

Reply:
Done